norddrop/ffi/bindings/norddrop.h

```diff
@@ -405,7 +401,7 @@ enum norddrop_result norddrop_purge_transfers_until(const struct norddrop *dev,
  *             "by_peer": false
  *         }
  *     ],
- *     "type": "outgoing",
```

Comment:
This was actually a bug, in the example, most probably the same entry from Outgoing was copied